packages/font-glyphs/src/space/index.ptl

```diff
@@ -71,3 +72,7 @@ glyph-block Spaces : begin
 		create-glyph [MangleName 'emsp'] [MangleUnicode 0x2003] : glyph-proc
 			set-width : if (para.isQuasiProportional && MosaicWidthScalar > 1) UPM MosaicWidth
 		alias [MangleName 'emquad'] [MangleUnicode 0x2001] [MangleName 'emsp']
+
```

Comment:
Please remove. It should follow the metrics of ideographs which Iosevka doesn't have.

Reply:
It would need to be added anyway if #759 is added. It is the missing space character in that block.